lib/pacemaker/pcmk_acl.c

```diff
@@ -41,8 +41,68 @@ static const xmlChar *NS_WRITABLE = (const xmlChar *) ACL_NS_PREFIX "writable";
 static const xmlChar *NS_READABLE = (const xmlChar *) ACL_NS_PREFIX "readable";
 static const xmlChar *NS_DENIED =   (const xmlChar *) ACL_NS_PREFIX "denied";
 
+/*!
+ * \brief This function takes a node and marks it with the namespace
+ *        given in the ns parameter.
+ *
+ * \param[in,out] i_node
+ * \param[in] ns
+ * \param[in,out] ret
+ * \param[in,out] ns_recycle_writable
+ * \param[in,out] ns_recycle_readable
+ * \param[in,out] ns_recycle_denied
+ */
+static void
+pcmk__acl_mark_node_with_namespace(xmlNode *i_node, const xmlChar *ns, int *ret, xmlNs **ns_recycle_writable, xmlNs **ns_recycle_readable, xmlNs **ns_recycle_denied)
+{
+    if (ns == NS_WRITABLE)
+    {
```

Comment:
fyi for the future, pacemaker style is for braces to go on the same line as if / else if